packages/tc-ui/src/primitives/temporal/server.jsx

```diff
@@ -25,19 +17,22 @@ const formatDateTime = (value, type) => {
 	}
 
 	if (type === 'Time') {
-		const timeInput = value;
-		return formatTime(timeInput);
+		return formatValue(`2020-01-15T${value}`, 'h:mm:ss a');
 	}
 };
 
 const convertValueForHTMLInput = (wrappedValue, type) => {
 	if (!(wrappedValue && wrappedValue.formatted)) return null;
 	const value = wrappedValue.formatted;
-	if (type === 'Time') return value;
+
```

Comment:
Just a question around this really, but if we have no use case might not be an answer. But I guess if people input through the UI we don't add a timezone? so if someone is adding a timezone  through the API then it would suggest it would be important to the user? so wouldn't it make more sense to add a default timezone based on user location or something  rather than strip the timezone?

Reply:
Trouble is that the `type="time"` input doesn't support displaying a time with a timezone. We didn't realise this though until after the API already implemented saving times with a default timezone (which I don't think was deliberate, it just sorta happened).

So in the longer term there's 1 of 2 paths to follow (depending on what our users actually want)
1. rewrite the API to never save timezones
2. rewrite the UI to support editing timezones

I don't really want to do either until we know what's required

Reply:
fair enough - that makes sense. 

What would the plan be to find out? or would you wait until it was mentioned in feedback from a user?